### PR TITLE
Added scripts to set up a Docker container w/ wikiedit-monitor running.

### PR DESCRIPTION
**Pull request by Barbarrosa**
@leebradley This switches to the library mentioned in https://github.com/leebradley/wikiedit-monitor/issues/20 and adds scripts for creating a development Docker container.

### CODE REVIEW

**Comment by ameliabradley** [comment]
Looks good, just a few notes:
1) Streamline instructions to just the docker instruction set
2) As we discussed, we need to add `run-script` to the npm command

File: README.md [hunk 1]
```diff
@@ -11,7 +11,34 @@ Project goals:
...
-## Setup
+## Development Setup
+Prerequisites:
+* Docker
+* Node.js
+
+1. Copy `(project-root)/conf/docker-config.json` to `(project-root)/config.json`.
+2. Start the MongoDB instance by running `npm mongo-docker`.
+3. The monitor and web interface by running `npm node-docker-win` (Windows) or `npm node-docker-nix` (Linux and OS X).
+4. Access the web interface at `http://localhost:8081`.
```
Comment (ameliabradley) [inline README.md]: I'm not sure if the local web interface would be accessible at this step if the user followed these instructions.

File: conf/docker-config.json.dist [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+      "port": 8082
+  },
+  "wsUrl": "ws://localhost:8082/",
+  "wsUiConfig": { "timeout": 250, "editLogEvents": {"wikiedits":1} },
+
+  "//": "For unit testing (WILL BE DROPPED AND RECREATED)",
+  "testConString": "mongodb://localhost:27017/wikimonitor-test",
+
+  "max_reconnection_attempts": 13
+}
```
Comment (ameliabradley) [inline conf/docker-config.json.dist:22]: What is the dist file for?

**Comment by ameliabradley** [comment]
Maybe instead of updating the npm scripts and whatnot, we should consider including a docker-compose.yml file?

See: https://docs.docker.com/compose/gettingstarted/

File: conf/docker-config.json.dist (hunk 2 above)
Comment (Barbarrosa) [reply conf/docker-config.json.dist:22]: This is mentioned (and used) in the new instructions. `.dist` is generally used for configuration files that should be copied and customized.

Also see:
http://stackoverflow.com/questions/16843080/what-does-dist-used-as-an-extension-of-some-source-code-file-mean
https://github.com/symfony/symfony/blob/master/phpunit.xml.dist
https://github.com/Seldaek/monolog/blob/master/phpunit.xml.dist
http://symfony.com/doc/current/best_practices/configuration.html

**Comment by Barbarrosa** [comment]
@leebradley I implemented your suggestion to use Docker Compose, and I tested to make sure that it works.

File: conf/docker-config.json.dist (hunk 2 above)
Comment (Barbarrosa) [reply conf/docker-config.json.dist:22]: I guess it wasn't mentioned... that's fixed now.

**Comment by ameliabradley** [comment]
Awesome 👍

**Pull request merged**